pallets/staking/src/lib.rs

```diff
@@ -2228,11 +2231,11 @@ decl_module! {
         /// - Write Each: System Account, Locks, Ledger (3 items)
         /// # </weight>
         #[weight =
-            120 * WEIGHT_PER_MICROS
+            ((120 * WEIGHT_PER_MICROS
             + 54 * WEIGHT_PER_MICROS * Weight::from(T::MaxNominatorRewardedPerValidator::get())
             + T::DbWeight::get().reads(7)
             + T::DbWeight::get().reads(5)  * Weight::from(T::MaxNominatorRewardedPerValidator::get() + 1)
-            + T::DbWeight::get().writes(3) * Weight::from(T::MaxNominatorRewardedPerValidator::get() + 1)
+            + T::DbWeight::get().writes(3) * Weight::from(T::MaxNominatorRewardedPerValidator::get() + 1)) * 25) / 100
```

Comment:
`/4` would've sufficed instead of `*25/100` :)

Reply:
Yeah but I keep this to be consistent with other divisions in our code base.

Reply:
I'm not sure which other divisions you mean. It's never a good idea to do `*25/100` instead of `/4`.